frontend/src/pages/cohorts/CohortsPage.tsx

```diff
@@ -1,74 +1,104 @@
+import { useRef, useState } from "react";
```

Comment:
something that I noticed got lost with the refactoring is the cohorts page "matching results" text

in prod you see something like 
```
253 matching cohorts (12,185 unique samples)
```

in dev it only says 
```
234 matches
```

iirc the unique sample count was something the PMs specifically asked for if you wouldn't mind adding that back 

while doing this might as well update the text for the patients, samples, and requests pages too

Reply:
Great catch. How do you feel about `matches` instead of `matching cohorts` here? I'm trying to avoid elements in the toolbar flowing into a second row, especially on the Samples page where we have filtering tabs, PHI toggle, and the changes buttons.

```
253 matches (12,185 unique samples)
``

Reply:
That works!